docs/installation-docker.md

```diff
@@ -0,0 +1,31 @@
+# Installation
+
+## Requirements
+
+docker + docker-compose
+
```

Comment:
Maybe we should add links to install Docker and Docker Compose.

Reply:
In my opinion if somebody is interested how to run docker he will have already installed it.

Reply:
Links to installation instructions would be nice. The cargo sample is a demo application so chances are that someone wants to learn new stuff and also try docker. Not really the scope of the project but if we communicate docker as an official way to run the demo then we should provide links how to set it up.